### ISSUE TITLE
writeNex5File.m bug -   maxTimestampTick = maxTimestampInTicks(nexFile)

### ISSUE THREAD

**Issue by k-mazurek** [arouseKUMC/WaveLimit#3]
In the writeNex5File.m function, my code errors out at this line 

`maxTimestampTick = maxTimestampInTicks(nexFile);`

Because I didn't know to what matTimestampsinTicks(nexFile) was referencing, I changed it to

`try
    maxTimestampTick = maxTimestampInTicks(nexFile);
catch
    maxTimestampTick = nexFile.tend;
end`

Now the function fully runs but wanted to check that was correct...

**Comment by arouseKUMC** [comment]
Hi Kevin,

writeNex5File.m was actually edited by Zheng.  I’d just been using the writeNexFile.  I had just added it to the repository but not really tested it.  (probably bad project management on my part)  I think Zheng is using a newer version of the HowToReadAndWriteNexAndNex5FilesInMatlab package.  You could try updating that as I can’t find a maxTimestampInTicks function in anything I have.  Otherwise, I would check with Zheng.  I’ll also try to get it running but may take me longer than if you check with him.  I know he was doing this as there’s some issues with larger files using the plain .nex file type.

-Adam

**Comment by k-mazurek** [comment]
gotcha, thanks! I figured as much since the variable was not listed in the writeNex.m file. Coincidentally, the OFS version I'm using I don't think can handle the nex5 files anyways so I think for now I'm just going to attempt to use the nex files and see how it goes. I'll leave this open for a little and see if Zheng sees it to see if he's had this error before or not.

**Comment by add-one-second** [comment]
Hi Kevin,
Yes, that is the case. I used HowToReadAndWriteNexAndNex5FilesInMatlab package released on Oct. 2019, which has the function 'maxTimestampInTicks'.
I used .nex5 files since some of my recordings are >2Gb, and .nex data only supports up to 2Gb (waveforms in some channels become incorrect).
I updated the HowToReadAndWriteNexAndNex5FilesInMatlab library, and it didn't change the behavior of the rest of the code, as far as I know.
Let me know if it still fails on you after updating the library.
